8.cytopick_analysis/scripts/feature_median_image_find.py

```diff
@@ -0,0 +1,363 @@
+print(median_index)
+print(
+    f"Median: {median}",
+    f"Image ID: {image_id}",
+    f"Cell ID: {cell_id}",
+    f"Well ID: {well_id}",
+    f"row_id: {row_id}",
+    f"fov_id: {fov_id}",
+    f"column id: {column_id}",
+    f"Center X: {center_x}",
+    f"Center Y: {center_y}",
+    f"Max X: {max_x_box}",
+    f"Max Y: {max_y_box}",
+    f"Min X: {min_x_box}",
+    f"Min Y: {min_y_box}",
+    sep="\n",
+)
+
+
+# In[7]:
+
+
+well_dict = {
+    "A": "01",
+    "B": "02",
+    "C": "03",
+    "D": "04",
+    "E": "05",
+    "F": "06",
+    "G": "07",
+    "H": "08",
+    "I": "09",
+    "J": "10",
+    "K": "11",
+    "L": "12",
+    "M": "13",
+    "N": "14",
+    "O": "15",
+    "P": "16",
+}
+column_dict = {
+    "1": "01",
+    "2": "02",
+    "3": "03",
+    "4": "04",
+    "5": "05",
+    "6": "06",
+    "7": "07",
+    "8": "08",
+    "9": "09",
+    "10": "10",
+    "11": "11",
+    "12": "12",
+    "13": "13",
+    "14": "14",
+    "15": "15",
+    "16": "16",
+    "17": "17",
+    "18": "18",
+    "19": "19",
+    "20": "20",
+    "21": "21",
+    "22": "22",
+    "23": "23",
+    "24": "24",
+}
+fov_dict = {
+    "1": "01",
+    "2": "02",
+    "3": "03",
+    "4": "04",
+    "5": "05",
+    "6": "06",
+    "7": "07",
+    "8": "08",
+    "9": "09",
+    "10": "10",
+    "11": "11",
+    "12": "12",
+    "13": "13",
+    "14": "14",
+    "15": "15",
+    "16": "16",
+}
+
+
+# In[8]:
+
+
+image_basename_1 = "p04-ch1sk1fk1fl1_IC.tiff"
+image_basename_2 = "p04-ch2sk1fk1fl1_IC.tiff"
+image_basename_3 = "p04-ch3sk1fk1fl1_IC.tiff"
+image_basename_4 = "p04-ch4sk1fk1fl1_IC.tiff"
+image_basename_5 = "p04-ch5sk1fk1fl1_IC.tiff"
+
+
+# In[9]:
+
+
+print(row_id, column_id, image_id, fov_id)
+print(f"r{well_dict[row_id]}c{column_dict[column_id]}f{fov_dict[fov_id]}")
+
+
+# In[10]:
+
+
+image_name1 = f"r{well_dict[row_id]}c{column_dict[column_id]}f{fov_dict[fov_id]}{image_basename_1}"
+image_path1 = image_dir_path.joinpath(image_name1)
+print(image_name1, "\n", image_path1)
+
+image_name2 = f"r{well_dict[row_id]}c{column_dict[column_id]}f{fov_dict[fov_id]}{image_basename_2}"
+image_path2 = image_dir_path.joinpath(image_name2)
+print(image_name2, "\n", image_path2)
+
+image_name3 = f"r{well_dict[row_id]}c{column_dict[column_id]}f{fov_dict[fov_id]}{image_basename_3}"
+image_path3 = image_dir_path.joinpath(image_name3)
+print(image_name3, "\n", image_path3)
+
+image_name4 = f"r{well_dict[row_id]}c{column_dict[column_id]}f{fov_dict[fov_id]}{image_basename_4}"
+image_path4 = image_dir_path.joinpath(image_name4)
+print(image_name4, "\n", image_path4)
+
+image_name5 = f"r{well_dict[row_id]}c{column_dict[column_id]}f{fov_dict[fov_id]}{image_basename_5}"
+image_path5 = image_dir_path.joinpath(image_name5)
+print(image_name5, "\n", image_path5)
+
+
+# In[11]:
+
+
+# define the radius of the circle to be drawn (actually a square) but a circle fits inside a square so it's all good
+# side of square - diameter of circle
+# radius = side / 2
+radius = 75
```

Comment:
Just curious, is there a reason why you chose this method over defining the box size (e.g., box size = 250 pixels)?